ens-names/src/keyer.rs

```diff
@@ -0,0 +1,7 @@
+pub fn ens_domain_key(asset_address: &String) -> String {
+    format!("domain:{}", asset_address)
```

Comment:
what does an `asset_address` refer to here?

Reply:
I have renamed this to `ens_hash`.